diff -ur a/src/GroupCell.cpp b/src/GroupCell.cpp
--- a/src/GroupCell.cpp	2020-04-18 08:14:02.000000000 +0100
+++ b/src/GroupCell.cpp	2020-06-02 18:21:17.184809273 +0100
@@ -1561,7 +1561,7 @@
   if (configuration->ShowCodeCells())
   {
     str += wxString::Format(
-      "\n\n\\noindent\n%%%%%%%%%%%%%%%\n%%% INPUT:\n\\begin{minipage}[t]{%iem}\\color{red}\\bfseries\n",
+      "\n\n\\noindent\n%%%%%%%%%%%%%%%%\n%%% INPUT:\n\\begin{minipage}[t]{%iem}\\color{red}\\bfseries\n",
       configuration->GetLabelWidth()/14
       ) + m_inputLabel->ToTeX() +
       wxString("\n\\end{minipage}");
@@ -1576,7 +1576,7 @@
 
   if (m_output != NULL)
   {
-    str += wxT("\n%%% OUTPUT:\n");
+    str += wxT("\n%%%% OUTPUT:\n");
     // Need to define labelcolor if this is Copy as LaTeX!
     if (imgCounter == NULL)
       str += wxT("\\definecolor{labelcolor}{RGB}{100,0,0}\n");
@@ -1637,7 +1637,7 @@
     {
       // Some invisible dummy content that keeps TeX happy if there really is
       // no output to display.
-      str += wxT("\\mbox{}\n\\]\n%%%%%%%%%%%%%%%");
+      str += wxT("\\mbox{}\n\\]\n%%%%%%%%%%%%%%%%");
     }
   }
   else
