diff -ur wxmaxima-Version-19.04.3.orig/src/Worksheet.cpp wxmaxima-Version-19.04.3/src/Worksheet.cpp
--- wxmaxima-Version-19.04.3.orig/src/Worksheet.cpp	2019-04-18 18:14:39.000000000 +0100
+++ wxmaxima-Version-19.04.3/src/Worksheet.cpp	2019-05-01 18:02:03.361174327 +0100
@@ -398,7 +398,6 @@
   dc.SetBackground(*(wxTheBrushList->FindOrCreateBrush(GetBackgroundColour(), wxBRUSHSTYLE_SOLID)));
   dc.SetMapMode(wxMM_TEXT);
   dc.SetBackgroundMode(wxTRANSPARENT);
-  PrepareDC(dc);
   
   // Create a graphics context that supports antialiassing, but on MSW
   // only supports fonts that come in the Right Format.
