From 20a6163d92bae2bfa569ffcefdbdd959e15abf22 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gunter=20K=C3=B6nigsmann?= <gunter@peterpall.de>
Date: Wed, 6 Mar 2019 12:08:28 +0100
Subject: [PATCH] Resolve the "maxima has finished calculating" on startup.

Resolves #1091
---
 src/wxMaxima.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wxMaxima.cpp b/src/wxMaxima.cpp
index 25e8a915..a38a15cb 100644
--- a/src/wxMaxima.cpp
+++ b/src/wxMaxima.cpp
@@ -7929,7 +7929,7 @@ void wxMaxima::TryEvaluateNextInQueue()
 
     // If the window isn't active we can inform the user that maxima in the meantime
     // has finished working.
-    if(m_worksheet->m_configuration->NotifyIfIdle())
+    if((m_worksheet->m_configuration->NotifyIfIdle()) && (m_worksheet->GetTree() != NULL))
       m_worksheet->SetNotification(_("Maxima has finished calculating."));
 
     return; //empty queue
